packages/frontend/src/providers/ExplorerProvider.tsx

```diff
@@ -1679,7 +1678,7 @@ export const ExplorerProvider: FC<
         if (unsavedChartVersion.metricQuery.sorts.length <= 0 && defaultSort) {
             setSortFields([defaultSort]);
         } else {
-            mutateAsync();
+            void mutateAsync();
```

Comment:
Should you just `return` this promise here and let the caller decide how to handle it?